daphne/cli.py

```diff
@@ -198,6 +221,23 @@ def _get_forwarded_port(self, args: Namespace):
         if args.proxy_headers:
             return "X-Forwarded-Port"
 
+    def _guess_asgi_protocol(self, application):
```

Comment:
Why not just use `asgiref.compatability` for this once we merge it? I'd rather not have it reimplemented everywhere.